test/jdk/java/awt/print/PrinterJob/PrintAllFonts.java

```diff
@@ -33,27 +33,25 @@
  * @test
  * @bug 4884389 7183516
  * @key printer
- * @library /test/lib /java/awt/regtesthelpers
+ * @library /java/awt/regtesthelpers
  * @build PassFailJFrame
  * @summary Font specified with face name loses style on printing
  * @run main/manual PrintAllFonts
  */
 public class PrintAllFonts implements Printable {
-
-    private static Font[] allFonts;
-    private int lineHeight = 18;
+    private static final int LINE_HEIGHT = 18;
+    private static final int FONT_SIZE = 14;
+    private final Font[] allFonts =
+            GraphicsEnvironment.getLocalGraphicsEnvironment().getAllFonts();
     private int fontNum = 0;
     private int startNum = 0;
     private int thisPage = 0;
-
```

Comment:
This blank was actually good, it separated the instance fields from the instructions constant.